drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -336,12 +336,23 @@ def alerts
       load_ar_association(object, :alerts, scope: Hmis::ClientAlert.active).sort_by(&:created_at).reverse
     end
 
+    # not optimized for batch queries, causes n+1 queries
     def hud_chronic
       return unless current_permission?(permission: :can_view_hud_chronic_status, entity: object)
 
-      # causes n+1 queries
-      enrollments = object.enrollments.where(data_source_id: current_user.hmis_data_source_id)
-      !!object.destination_client&.as_warehouse&.hud_chronic?(scope: enrollments)
+      open_enrollments = object.enrollments.open_excluding_wip.preload(:ch_enrollment, :project)
+
+      # Check if there's an open enrollment at a residential project with a move in date that has occurred
+      today = Date.current
+      client_is_housed = open_enrollments.any? do |enrollment|
+        enrollment.project.project_type.in?(HudUtility2024.residential_project_type_ids) && (enrollment.move_in_date&.<= today)
```

Comment:
I think this is too broad. I probably incorrectly said to check for "move-in date on residential project" earlier, forgetting that ES and SO and others are considered residential in the reporting world. That always trips me up.

Move-in Date is only collected for Permanent Housing, so I believe we should limit this instead to: `project_type.in?(HudUtility2024.permanent_housing_project_types)`


for reference:
```ruby
> HudUtility2024.residential_project_type_ids
=> [0, 1, 2, 3, 4, 8, 9, 10, 13]
#.. aka
> HudUtility2024.residential_project_type_ids.map { |id| HudUtility2024.project_type(id) }
=>
["Emergency Shelter - Entry Exit",
 "Emergency Shelter - Night-by-Night",
 "Transitional Housing",
 "PH - Permanent Supportive Housing",
 "Street Outreach",
 "Safe Haven",
 "PH - Housing Only",
 "PH - Housing with Services (no disability required for entry)",
 "PH - Rapid Re-Housing"]
```

vs
```ruby
> HudUtility2024.permanent_housing_project_types
=> [3, 9, 10, 13]
> HudUtility2024.project_types_with_move_in_dates
=> [3, 9, 10, 13]
> HudUtility2024.permanent_housing_project_types.map { |id| HudUtility2024.project_type(id) }
=>
["PH - Permanent Supportive Housing",
 "PH - Housing Only",
 "PH - Housing with Services (no disability required for entry)",
 "PH - Rapid Re-Housing"]
```